### PR TITLE
fix: production compose database healthcheck user

### PR DESCRIPTION
**Pull request by c3ppc3pp**
This PR fixes database healthchecks in global docker compose files. Also fixes some formatting, and minor things.

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -173,31 +165,27 @@ services:
...
-
-  # This is insecure, do not use this in production or on the public internet
-  # mailslurper:
-  #   image: docker.io/oryd/mailslurper:smtps-latest
-  #   ports: # port 1025 is used internally for smtp, you do not have to expose that
-  #     - 4436:4436 # web UI
-  #     - 4437:4437 # http API
-  #   restart: unless-stopped
-
+      # This is insecure, do not use this in production or on the public internet
```
Comment (m8vago) [inline docker-compose.yaml:182]: this indent is faulty

File: docker-compose.yaml [hunk 2]
```diff
@@ -1,31 +1,28 @@
+---
 # version is removed as per: https://github.com/docker/cli/pull/3445
```
Comment (polaroi8d) [inline docker-compose.yaml:2]: Could you remove this comment? I think it's obsoleted, rn.

File: docker-compose.yaml (hunk 1 above)
Comment (polaroi8d) [reply docker-compose.yaml:182]: Was this formatted using prettier? I believe the root problem is that you haven't set up prettier properly.

Comment (c3ppc3pp) [reply docker-compose.yaml:186]: will fix in later PR

File: docker-compose.yaml (hunk 2 above)
Comment (c3ppc3pp) [reply docker-compose.yaml]: Fixed.

**Pull request merged**